### PR TITLE
(SIMP-2527) Update rsyslog calls

### PR DESCRIPTION
**Pull request by jeefberkey**
- Cleaned up rsyslog calls to remove logic

SIMP-2527 #close

### CODE REVIEW

File: manifests/server.pp [hunk 1]
```diff
@@ -63,37 +63,35 @@
...
-  if ($facts['os']['name'] in ['RedHat', 'CentOS']) and (versioncmp($facts['os']['release']['major'], '7') < 0) {
-    file { '/etc/init.d/rsyncd':
-      ensure  => 'file',
-      owner   => 'root',
-      group   => 'root',
-      mode    => '0750',
-      content => file("${module_name}/rsync.init")
-    }
-
+  if 'systemd' in $facts['init_systems'] {
```
Comment (nick-markowski) [inline manifests/server.pp]: shouldn't this be the el6 case, if 'systemd' not in init_system?

Comment (jeefberkey) [inline manifests/server.pp]: I flipped the content around, and made systemd the special case, not the default case.

Comment (nick-markowski) [inline manifests/server.pp]: diffs are hard.

**Comment by jeefberkey** [comment]
Works on a live system.

**Pull request merged**